etc/tests/test_integration.py

```diff
@@ -385,8 +388,14 @@ def clean_up_program_htsget(program_id):
 
 
 def test_ingest_not_admin_katsu():
-    clean_up_program("SYNTHETIC-1")
-    clean_up_program("SYNTHETIC-2")
+    katsu_response = requests.get(f"{ENV['CANDIG_ENV']['KATSU_INGEST_URL']}/v2/discovery/programs/").json()
```

Comment:
do you need to check to make sure that this returns a json-able response? what if it's not status_code == 200?